src/Tizen.NUI.Components/Controls/MenuItem.cs

```diff
@@ -66,31 +64,6 @@ protected override void Dispose(DisposeTypes type)
             base.Dispose(type);
         }
 
-        /// <summary>
-        /// Applies style to MenuItem.
-        /// </summary>
-        /// <param name="viewStyle">The style to apply.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public override void ApplyStyle(ViewStyle viewStyle)
```

Comment:
Just question! 
Why do you remove ApplyStyle() override method here?
No need to exist anymore?

Reply:
It was added to check "styleApplied" in methods.
But all the methods are removed in this PR.
So yes, it is not required to exist any more.